test/Directory.Build.targets

```diff
@@ -1,5 +1,13 @@
 <Project>
 
   <Import Project="..\Directory.Build.targets" />
-
+  
+  <PropertyGroup Condition="'$(Coverage)' == 'true' and '$(Configuration)'=='Release' and '$(OS)' == 'Windows_NT' and '$(TargetArchitecture)' == 'x64'">
+      <CollectCoverage>true</CollectCoverage>
+      <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+      <CoverletOutputFormat>opencover</CoverletOutputFormat>
+      <CoverletOutput>$(MSBuildThisFileDirectory)..\bin\coverage\$(AssemblyName).xml</CoverletOutput>
```

Comment:
Use `$(BaseOutputPath)` instead of explicitly pathing to it. We have "common" properties for these directories [https://github.com/dotnet/machinelearning/blob/d26510f5efc6a22f0d1a73663e22095f60a4bcf6/Directory.Build.props#L30-L40](https://github.com/dotnet/machinelearning/blob/d26510f5efc6a22f0d1a73663e22095f60a4bcf6/Directory.Build.props#L30-L40)

Shouldn't we be putting these files under `$(BaseOutputPath)$(PlatformConfig)` ? That way I can run debug vs. release, and the files don't overwrite each other.